aria/modeling/service_instance.py

```diff
@@ -1685,12 +1725,31 @@ def inputs(cls):
     # endregion
 
     description = Column(Text)
+    relationship_edge = Column(Boolean)
     implementation = Column(Text)
+    configuration = Column(modeling_types.StrictDict(key_cls=basestring))
     dependencies = Column(modeling_types.StrictList(item_cls=basestring))
     executor = Column(Text)
     max_retries = Column(Integer)
     retry_interval = Column(Integer)
 
+    def configure(self):
+        from . import models
+        if (self.implementation is None) or (self.interface is None):
```

Comment:
It will be None if the operation is attached to `Service` as a workflow :) I know you suggested a different model entirely for workflows ... that would be a separate JIRA perhaps.